src/Widgets/PagesList.vala

```diff
@@ -118,6 +118,7 @@ public class ENotes.PagesList : Gtk.Box {
         plus_button.can_focus = false;
         minus_button.no_show_all = true;
         separator.no_show_all = true;
+        minus_button.set_tooltip_text (_("Delete Page"));
```

Comment:
This should be moved to it's own PR.

A single PR should only change/add one feature at a time :)

Reply:
Alright, I didn't know about that :/

Reply:
No problem :) Just helping you learn the best practices for when you contribute in another code